cmd/neutrond/root.go

```diff
@@ -87,7 +88,14 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 			}
 
 			customTemplate, customNeutronConfig := initAppConfig()
-			return server.InterceptConfigsPreRunHandler(cmd, customTemplate, customNeutronConfig, tmcfg.DefaultConfig())
+			err = server.InterceptConfigsPreRunHandler(cmd, customTemplate, customNeutronConfig, tmcfg.DefaultConfig())
```

Comment:
You can just create our own `DefaultConfig()` instead of `tmcfg.DefaultConfig()` with `timeout_commit = 5s`